### PR TITLE
[MOD-40] Add Google Analytics script to HTML head

### PR DESCRIPTION
**Pull request by enymuss**
Wasn't sure about the desciption of the task and when with my best judgemnt. The script appears in the HTML but I wasn't able to see the results while running on my localhost.

screen shot 2015-01-16 at 22 25 40

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -74,6 +87,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-watch')
   grunt.loadNpmTasks('grunt-contrib-connect')
   grunt.loadNpmTasks('grunt-colorguard')
+  grunt.loadNpmTasks('grunt-html-build')
```
Comment (enymuss) [inline Gruntfile.js]: Link: https://github.com/spatools/grunt-html-build

File: app/js/googleAnalytics.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<script>
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+ga('create', 'UA-58652123-1', 'auto');
```
Comment (enymuss) [inline app/js/googleAnalytics.js]: 'UA-58652123-1' - this code is connected with my google account. Has to be changed.

**Comment by dkayiwa** [comment]
@elliottwilliams what do you think of this pull request?

File: app/js/googleAnalytics.js (hunk 2 above)
Comment (dkayiwa) [inline app/js/googleAnalytics.js]: @enymuss so are you planning to change the above?

**Comment by dkayiwa** [comment]
@enymuss do you mind squashing these commits into one?

**Comment by dkayiwa** [comment]
@enymuss i can reopen this if you still have time to complete it. :)

**Pull request closed without merging**